### PR TITLE
M0090IDE-4514 Add a tutorial on Offline Repository in SDK 6

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
https://docs.microej.com/en/feature-m0090ide-4514_offline-repository/

### CODE REVIEW

**Review by denisbedin — APPROVED** [review]

File: SDK6UserGuide/tutorials/offlineRepository.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Creating and Using an Offline Repository
+========================================
+
+Developing MicroEJ projects requires the Gradle plugins used for the build, 
+as well as the modules (Add-On Libraries, Foundation Libraries, ...) used by the project code.
+All these artifacts must be available in artifact repositories.
+
+MicroEJ provides them as online repositories which can be used directly, 
+thanks to the configuration described in the :ref:`sdk_6_configure_repositories` section.
+However it is not always possible to rely on these online repositories.
```
Comment (emoreac-microej) [inline SDK6UserGuide/tutorials/offlineRepository.rst:12]: However,

File: SDK6UserGuide/tutorials/offlineRepository.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+If this online repository, or another one, contains all the module required for your project, download it. 
+For example for the Central Repository, go to `its location <https://forge.microej.com/ui/repos/tree/General/microej-central-repository-release>`__ 
+and click on the :guilabel:`Download` button.
+
+Now go to :ref:`this section <sdk_6_use-offline-modules-repository>` to configure your project to use it.
+
+Custom Offline Repository
+#########################
+
+If you need to have a custom Offline Repository (for example because the available online repositories
```
Comment (emoreac-microej) [inline SDK6UserGuide/tutorials/offlineRepository.rst:92]: If you need to have a custom Offline Repository -> If you need a custom Offline Repository

File: SDK6UserGuide/tutorials/offlineRepository.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            setM2compatible(true)
+        }
+    }
+
+    ...
+
+  }
+
+..
+   | Copyright 2008-2023, MicroEJ Corp. Content in this space is free
```
Comment (emoreac-microej) [inline SDK6UserGuide/tutorials/offlineRepository.rst:135]: | Copyright 2008-2024

**Review by emoreac-microej — COMMENTED** [review]
The content is ok for me, only few "typos" I noticed.

File: SDK6UserGuide/tutorials/offlineRepository.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/offlineRepository.rst]: Done

File: SDK6UserGuide/tutorials/offlineRepository.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/offlineRepository.rst]: Done

File: SDK6UserGuide/tutorials/offlineRepository.rst (hunk 3 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/offlineRepository.rst]: Done

**Review by emoreac-microej — APPROVED** [review]

**Pull request merged**